owncloudComLibrary/build.gradle

```diff
@@ -15,6 +15,13 @@ dependencies {
         exclude module: "kotlin-reflect"
     }
     kapt "com.squareup.moshi:moshi-kotlin-codegen:$moshiVersion"
+
+    testImplementation "junit:junit:4.13"
+    androidTestImplementation 'androidx.test.espresso:espresso-core:3.1.1'
+    androidTestImplementation 'androidx.test:runner:1.1.1'
+    androidTestImplementation 'androidx.test:core:1.1.0'
+    androidTestImplementation 'androidx.test.ext:junit:1.1.0'
+    androidTestImplementation 'androidx.test:rules:1.1.1'
```

Comment:
About this, i think that most of this dependencies are not needed. Could you try to run test just with `androidx.test:runner:1.1.1` one? 

BTW, try to include updated dependecies 😉

Reply:
done